easybuild/easyblocks/p/parmetis.py

```diff
@@ -182,9 +182,9 @@ def install_step(self):
     def sanity_check_step(self):
         """Custom sanity check for ParMETIS."""
 
-        # Paths to libraries depend on SHARED configuration option
-        config_trues = ['1', 'ON', 'YES', 'TRUE', 'Y']  # strings that evaluate as True in CMake
-        if set(['-DSHARED=%s' % opt for opt in config_trues]) & set(self.cfg['configopts']):
+        # Paths to libraries defined depending on SHARED configuration option
+        config_shared = ['-DSHARED=%s' % val for val in ['1', 'ON', 'YES', 'TRUE', 'Y']]  # True values in CMake
```

Comment:
Lower case values are also allowed if i remember correctly? And "True", "Yes", "On", so a lower/upper case compare is likely better.

Reply:
You're right. I made the comparison case insensitive :+1: